Sources/Views/EpisodePage.swift

```diff
@@ -174,7 +174,7 @@ private func downloadsView(codeSampleDirectory: String) -> Node {
         .href(gitHubUrl(to: GitHubRoute.episodeCodeSample(directory: codeSampleDirectory))),
         .class([Class.pf.colors.link.yellow, Class.margin([.mobile: [.left: 1]]), Class.align.middle])
       ],
-      .text("\(codeSampleDirectory).playground")
+      .text("\(codeSampleDirectory)")
```

Comment:
I think we can even pass this directly now!
```suggestion
      .text(codeSampleDirectory)
```

Reply:
You're right 😅 sorry, it was a long day ^^